### PR TITLE
modified the Abortion Information partial view to include funding information

### LINKED ISSUE #450: Find some way to represent the primary contact so CMs know who they're talking to
From user testing in #405 -- 

> Add a field to specify the relationship of the primary contact to the patient.  And/or add a text field for explanation.  CMs want to understand who it is they are talking to.  [2 users]

Throwing it up to UX on how to represent this.

See also #402 for part of the technical changes here.

### LINKED ISSUE #480: Set up abortion cost section
Here's what's in the designs:

image

We need to flesh out the stuff on the right side and pop it into the abortion information partial. This will  entail integrating pledge objects into the form, so I'm marking backend. 

Wire up so it matches the designs, minus 'Add other funding source'. We can cross that bridge later. 

It's urgent that we get this in before Monday, so please make a PR for this at the end of the night no matter where you are in the process if you claim it.

### PR DESCRIPTION
**Pull request by dfelsent**
I rule and have completed some work on Case Manager that's ready for review!

This pull request makes the following changes:
* Fleshes out secondary contact fields; adds secondary_relationship attr
* Adds stubbed pledge fields to pregnancy model; similarly adds fields in form

It relates to the following issue #s: 
* Fixes #480 
* Fixes #450 

…ormation, per issue #480

### CODE REVIEW

File: app/models/pregnancy.rb [hunk 1]
```diff
@@ -45,6 +45,9 @@ class Pregnancy
   # Procedure result - generally for administrative use
   field :fax_received, type: Boolean
   field :procedure_cost, type: Integer
+  field :patient_contribution, type: Integer
```
Comment (colinxfleming) [inline app/models/pregnancy.rb]: Can you add a comment after these three fields to make a note that they're temporary stand-ins for `pledge` objects? Something like `field :patient_contribution, type: Integer # temporary` will do nicely!

File: app/views/pregnancies/_pledge.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 		<div class="pledge_form">
 			<div><span class="pledge_modal_category">Patient name:</span> <%= pregnancy.patient.name %></div>
 			<div><span class="pledge_modal_category">Patient ID:</span> <%= pregnancy.patient.primary_phone %></div>
-			<div><span class="pledge_modal_category">Pledge amount:</span> $100</div>
+			<div><span class="pledge_modal_category">Pledge amount:</span> $<%= pregnancy.dcaf_soft_pledge %> </div>
```
Comment (colinxfleming) [inline app/views/pregnancies/_pledge.html.erb]: 💯

File: app/views/pregnancies/_patient_information.html.erb [hunk 3]
```diff
@@ -9,11 +9,8 @@
 
       <div class="row">
         <div class="info-form-left col-sm-6">
-          <%= f.fields_for pregnancy.patient do |p| %>
-            <%= p.text_field :secondary_person, autocomplete: 'off' %>
-            <%= p.text_field :secondary_phone, autocomplete: 'off' %>
-          <% end %>
-    
+          <strong>First and last name:</strong><br><%= pregnancy.patient.name %><br>
```
Comment (colinxfleming) [inline app/views/pregnancies/_patient_information.html.erb]: Can you revert these changes? We want patient name and primary phone to still be editable.

File: app/views/pregnancies/_patient_information.html.erb [hunk 4]
```diff
@@ -38,6 +35,13 @@
           </div>
 
           <%= f.text_field :zip, label: 'ZIP', autocomplete: 'off' %>
+          
+          <h3> Secondary Contact</h3>
```
Comment (colinxfleming) [inline app/views/pregnancies/_patient_information.html.erb]: 💯

**Comment by colinxfleming** [comment]
Hey @dfelsent -- this looks pretty good! Thanks for punching it in! I had a few comments, let me know what you think.

Also wanted to ask you about the switch from h4 to h3 tags -- what's going on there? Want to make sure we have it documented so we don't go back and forth!

You're also going to want to add `secondary_relationship` to the permitted parameters in `pregnancies_controller`, so we can save it. 

Finally: The model changes et al are failing tests. You'll want to make a few additional changes to get the suite back to green. The failing tests and my suggested fixes are below.

```
# looks like this one is called `Secondary contact first and last name` now? Change the find and it should be fine
ERROR["test_0001_should alter the information", #<Class:0x00000006a63d98>, 20.342155362000995]
 test_0001_should alter the information#UpdatePatientInfoTest::changing patient information (20.34s)
Capybara::ElementNotFound:         Capybara::ElementNotFound: Unable to find field "Secondary person"
            test/integration/update_patient_info_test.rb:69:in `block (2 levels) in <class:UpdatePatientInfoTest>'

# change phone number back to an editable field to resolve this
 FAIL["test_0001_should alter the information", #<Class:0x00000006e2c060>, 24.36530392200075]
 test_0001_should alter the information#UpdatePatientInfoTest::changing patient dashboard information (24.37s)
        Expected false to be truthy.
        test/integration/update_patient_info_test.rb:37:in `block (3 levels) in <class:UpdatePatientInfoTest>'
        test/integration/update_patient_info_test.rb:32:in `block (2 levels) in <class:UpdatePatientInfoTest>'

# change this so it's looking for whatever the label for secondary phone is
 FAIL["test_0003_should show patient information on open", #<Class:0x00000006a7c050>, 30.934557238000707]
 test_0003_should show patient information on open#ShowPregnancyInformationTest::clicking between sections to see information (30.94s)
        Expected false to be truthy.
        test/integration/show_pregnancy_information_test.rb:41:in `block (3 levels) in <class:ShowPregnancyInformationTest>'
        test/integration/show_pregnancy_information_test.rb:39:in `block (2 levels) in <class:ShowPregnancyInformationTest>'

# add secondary_relationship to the hash
 FAIL["test_0003_should track proper info", #<Class:0x00000006e54a60>, 46.10000197100089]
 test_0003_should track proper info#AuditTrailTest::natural initializing - everything okay alarm (46.10s)
        --- expected
        +++ actual
        @@ -1 +1 @@
        -["name", "primary_phone", "secondary_person", "secondary_phone", "secondary_relationship", "updated_by_id"]
        +["name", "primary_phone", "secondary_person", "secondary_phone", "updated_by_id"]
        test/models/audit_trail_test.rb:25:in `block (2 levels) in <class:AuditTrailTest>'
```

**Comment by colinxfleming** [comment]
Let me know if you hit any snags or need a hand pushing any of this over the finish line. (I'll push it over the line on my own on Saturday night late if timing is an issue.)

Thanks again! This is one of the key final components and I'm very grateful.

**Comment by colinxfleming** [comment]
Pushing this over the finish line shortly. Thanks @dfelsent !

File: app/views/pregnancies/_patient_information.html.erb (hunk 3 above)
Comment (colinxfleming) [inline app/views/pregnancies/_patient_information.html.erb]: I think I misinterpreted this in a sleep deprived state -- thought this was moving name and primary phone down into another section, but I see it's not. Gonna pop these out for the time being and slightly edit the labels down in the secondary section.

**Comment by dfelsent** [comment]
Quite sorry, @colinxfleming  -- I was gone most of the weekend. Were you able to get to everything?
